frontend/src/components/worksheets/items/TableItem/BundleRow.js

```diff
@@ -347,7 +347,7 @@ class BundleRow extends Component {
                         <TableCell colSpan='100%' classes={{ root: classes.insertPanel }}>
                             <div className={classes.insertBox}>
                                 <NewRun
-                                    after_sort_key={this.props.after_sort_key}
+                                    after_sort_key={this.props.after_sort_key + 1} // insert the new run below the bundle row, so add 1 to after_sort_key
```

Comment:
Can you move this comment to the "Rerun" comment above (line 325), so it's clear that this comment applies to this entire section?

Reply:
updated.